raveandroid/src/main/java/com/flutterwave/raveandroid/banktransfer/BankTransferFragment.java

```diff
@@ -234,12 +234,6 @@ public void onClick(View view) {
         });
     }
 
-    @Override
```

Comment:
@jeremiahVaris Can you explain this?

Reply:
Wanted to persist bank transfer details in SavedInstanceState, but it was causing crashes for other fragments when there's no data loaded for the bank transfer, plus it wasn't solving the problem I had in mind, so I removed it.